src/containers/Layout/Layout.js

```diff
@@ -8,31 +8,77 @@ import Sidebar from "../../components/Sidebar/Sidebar";
 import Jokes from "../Jokes/Jokes";
 
 class Layout extends Component {
-  constructor(props){
-    super(props);
+  constructor(props) {
+    super(props)
 
     this.state = {
       jokesList: [],
+      numberOfJokes: 1
     }
-
+    
+    this.incrementHandler = this.incrementHandler.bind(this);
+    this.decrementHandler = this.decrementHandler.bind(this);
+    this.fetchRandomJoke = this.fetchRandomJoke.bind(this);
     this.handleClick = this.handleClick.bind(this);
-  }
+    }
 
-  handleClick(event) {
-    const { name } = event.target;
+    componentDidMount() {
+      this.fetchRandomJoke();
+    }
+  
+    incrementHandler() {
+      let incrementCounter = this.state.numberOfJokes;
+      incrementCounter++;
+      this.setState({numberOfJokes: incrementCounter})
```

Comment:
in `this.setState(...)` you could pass a function instead of object - so it could be something like `this.setState(prevState => ({numberOfJokes: ++prevState.numberOfJokes}))`. First argument is previous state and second one is props